packages/react-component-library/src/components/DatePickerE/DatePickerE.tsx

```diff
@@ -199,16 +216,17 @@ export const DatePickerE: React.FC<DatePickerEProps> = ({
             aria-owns={contentId}
             ref={inputButtonRef}
             type="button"
-            onClick={open ? handleOnClose : handleOnFocus}
+            onClick={open ? handleOnClose : handleOnOpen}
             disabled={isDisabled}
             data-testid="datepicker-input-button"
           >
-            <StyledSeparator />
-            <DropdownIndicatorIcon isOpen={open} />
+            <StyledIconCalendarWrapper>
```

Comment:
Just noticed `StyledIconCalendarWrapper` is misnamed – will rename to `StyledIconEventWrapper`.